modules/temp-deck/temp-deck-arduino/temp-deck-arduino.ino

```diff
@@ -301,16 +301,23 @@ void update_led_display(boolean debounce=true){
 
 void read_thermistor_and_apply_offset() {
   if (thermistor.update()) {
-    CURRENT_TEMPERATURE = thermistor.temperature();
+    double _current_temp;
+    _current_temp = thermistor.temperature();
```

Comment:
let's just initialize in the same line like `double current_temp = thermistor.temperature();`

also `_current_temp` makes me think this is a private class member; maybe just `current_temp`?